src/components/colorscale/attributes.js

```diff
@@ -13,6 +13,8 @@ module.exports = {
         valType: 'boolean',
         role: 'info',
         dflt: true,
+        editType: 'docalc',
+        impliedEdits: {zmin: undefined, zmax: undefined},
```

Comment:
Same with impliedEdits. I assume these are attributes that are automatically applied within the same attr path, but I'm unable to find where it's implemented and it's not immediately clear what the `^` prefix means.

Reply:
Never mind. Github was suppressing the plot_api diff. Found it.